### PR TITLE
misc: add confirmation to archive-branches script

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@protonmail.ch>

### CODE REVIEW

**Comment by Kubuxu** [comment]
Does anyone see branch that shouldn't be moved to an archive in this list:
```
exp/sync-concurrency
feat/bitswap/event-log
feat/buffer-bitswap
feat/cl
feat/cmds0.5
feat/commands2.0
feat/commands2.0-CR-cleanup
feat/coreapi-for-real-this-time
feat/coreapi-gateway2
feat/corehttp-check-api-version
feat/dep
feat/extract-dagservice
feat/fetcher-interfaces
feat/get-tar-stdout
feat/go-get-meta-tag
feat/mfs-fuse
feat/provider-strategies-1
feat/recurring-metrics
feat/update-ws-fixes
fix/1515
fix/add-panic-ctrl-d
fix/add-panic-ctrl-d-2
fix/coreunix/add-encod-json
fix/files/raw-leaves
fix/flaky-sharness-api-file
fix/gateway-flatfs-debug
fix/gc/internal
fix/publisher-tests
gx/update-06fz0m
gx/update/go-log
ipns-pubsub
jenkinsfile
keks-patch-1
kevina/kill-get-offline
kevina/statlocal-fix1
kevina/statlocal-fix2
```
Please give :+1: after you check and rise a comment if there is something that shouldn't be moved.

File: bin/archive-branches.sh [hunk 1]
```diff
@@ -48,9 +48,20 @@ active_branches() {
 
 git remote add archived "git@github.com:$org/$arch_repo.git" || true
 
-cat <(active_branches) <(pr_branches) <((IFS=$'\n'; echo "${exclusions[*]}")) \
-	| sort -u | comm - <(origin_refs | sort) -13 |\
-	while read -r ref; do
+branches_to_move="$(cat <(active_branches) <(pr_branches) <((IFS=$'\n'; echo "${exclusions[*]}")) | sort -u | comm - <(origin_refs | sort) -13)"
+
+echo "Please confirm move of following branches [y/N]:"
```
Comment (magik6k) [inline bin/archive-branches.sh]: I'd echo this after printing the list

**Comment by Kubuxu** [comment]
@kevina can you take a look?

**Comment by Kubuxu** [comment]
Enough time elapsed. I will move branches to archive after this is merged.

**Pull request merged**